app/src/processing/app/Base.java

```diff
@@ -1458,6 +1486,16 @@ public void actionPerformed(ActionEvent actionevent) {
     }));
     boardsCustomMenus.add(boardMenu);
 
+    // Insert recently used boards menu and remember index for insertion later
+    if (PreferencesData.has("recent.boards")) {
```

Comment:
This is a good point. In my head, it seemed ugly to leave a menu header without any entries underneath but you make a good point about there always being a board selected. This could become a check for `recent.num_boards == 0`